interop/quic-openssl-docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM martenseemann/quic-network-simulator-endpoint:latest
+
+# Make sure curl picks up the new openssl
+ENV PKG_CONFIG_LIBDIR=/usr/lib64/pkgconfig:/usr/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig/:$PKG_CONFIG_LIBDIR
+# Set the environment variable LD_LIBRARY_PATH to ensure we get the right libraries
+ENV LD_LIBRARY_PATH=/usr/lib64:/usr/lib:/usr/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+
+# Install needed tools
+RUN apt-get update && apt-get install -y \
```

Comment:
Just fyi, I believe the recommended tool for Ubuntu is `apt` instead of `apt-get`. But it probably doesn't matter in this case

Reply:
I think `apt-get` is the right choice here, because `apt` is more suitable for interactive use.
```
sashan@ubuntu:~$ apt
apt 2.4.12 (amd64)
Usage: apt [options] command

apt is a commandline package manager and provides commands for
searching and managing as well as querying information about packages.
It provides the same functionality as the specialized APT tools,
like apt-get and apt-cache, but enables options more suitable for
interactive use by default.
```